### PR TITLE
add TEMPLATE folder csv's

### PR DESCRIPTION
**Pull request by Piranias**
Fix xxx

**Changes proposed in this pull request**:
- Add Template folder for csv's with comments

The following steps were realized, as well (if applies):
- [ ] Use in-line comments to explain your code
- [ ] Write docstrings to your code
- [ ] For new functionalities: Explain in readthedocs
- [ ] Write test(s) for your new patch of code
- :x: Update the CHANGELOG.md (let's start this after the first release)
- [ ] Apply black (`black . --exclude docs/`)

*Please mark above checkboxes as following:*
- [ ] Open
- [x] Done

:x: Check not applicable to this PR

<sub>*For more information on how to contribute check the [CONTRIBUTING.md](https://github.com/rl-institut/mvs_eland/blob/dev/CONTRIBUTING.md).*<sub>

### CODE REVIEW

**Review by SabineHaas — CHANGES_REQUESTED** [review]
Thank you @Piranias!
For me the structure looks fine. I would like to ask @MaGering and @c-moeller for their feedback, as well. 

I'd suggest to name the folder `PARAMETERS_mvs_inputs` as a template would rather be something you can directly use (like `mvs_inputs_template_sector_coupling`)

**Comment by Piranias** [comment]
sure. also I did not fill in all the comments, I though we could do that all together :) So this is just the structure

**Comment by Piranias** [comment]
Hey, 
unfortunately I forgot that this is not pushed yet and I continued working on this branch. That's why I would like to push this now, I'll add another PR with the name change of the folder right after. sorry for the mix-up

**Comment by Piranias** [comment]
> Thank you @Piranias!
> For me the structure looks fine. I would like to ask @MaGering and @c-moeller for their feedback, as well.
> 
> I'd suggest to name the folder `PARAMETERS_mvs_inputs` as a template would rather be something you can directly use (like `mvs_inputs_template_sector_coupling`

File: pvcompare/data/TEMPLATE_mvs_inputs/csv_elements/energyConversion.csv [hunk 1]
```diff
@@ -0,0 +1,17 @@
+,unit,storage_charge_controller_in,storage_charge_controller_out,solar_inverter_01,comments storage_charge_controller in,comments storage_charge_controller_out,comments solar_inverter_01
```
Comment (c-moeller) [inline pvcompare/data/TEMPLATE_mvs_inputs/csv_elements/energyConversion.csv:1]: Does is make sense to have the same structure here as in energyConversion.csv? That means, all technologies one below the other in extra tables?

File: pvcompare/data/TEMPLATE_mvs_inputs/csv_elements/fixcost.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
+,unit,distribution_grid,engineering,operation
```
Comment (c-moeller) [inline pvcompare/data/TEMPLATE_mvs_inputs/csv_elements/fixcost.csv:1]: Same here? One below the other?

**Pull request merged**